cfgov/cfgov/urls.py

```diff
@@ -413,7 +422,7 @@
     except ImportError:
         pass
 
-if settings.DEPLOY_ENVIRONMENT != 'build':
+if settings.DEPLOY_ENVIRONMENT not in ['build', 'beta']:
```

Comment:
I'm going to see if using https://github.com/cfpb/wagtail-flags/pull/13 and an `include()` for old/new Ask URLs removes the need for this check here.